bsip-0083.md

```diff
@@ -0,0 +1,74 @@
+
+# Specification
+The blockchain software is designed to support assets with multisignature issuer authorities; therefore, the technical changes required for this proposal are straightforward. The following technical changes are required:
+
+**Assignment of Accumulated Fees**
+
+At the hardfork date, the accumulated fees of the existing BitAssets will be transferred to the committee-account.
+
+**Update to Committee-Controlled BitAsset Issuer Fields**
+
+At the hardfork date, the committee-controlled BitAsset objects' issuer fields will be updated to specify the cn-vote account (ID 1.2.1148184). The existing price feeds will be cleared, and the cn-vote account will be specified as the feed producer. After the hardfork, the cn-vote account will be able to assign other feed producers at its discretion.
+
+**Disable the witness_fed_asset and committee_fed_asset Flags**
+
+At the hardfork date, the `witness_fed_asset` and `committee_fed_asset` flags will be unset from the BitAssets, and will be disabled from use in future assets.
+
+**Amend total_core_in_orders to Track Limit and Collateral Orders Separately**
+
+The `total_core_in_orders` field of the `account_statistics_object` will be deprecated, and assets in limit orders will be tallied in a new `total_core_in_limit_orders` field while collateral will be tracked in a new `total_core_in_collateral_orders` map, keyed on the collateralized asset ID. During vote weight calculation, the `total_core_in_limit_orders` will add directly to the account’s voting weight; additionally, the `total_core_in_collateral_orders` which exceeds the feed price value of the collateralized debt will be added to the account’s voting weight as well.
+
+**Enable Remittance of Worker Proposals in Core Asset**
+
+Escrow agents administering existing worker proposals presently denominated in BitAssets shall be authorized to remit worker payments in BitShares core asset, or in a BitAsset selected at the discretion of the workers and escrow agents.
+
+# Discussion
+### Voting Power
+The voting power of debt holders will be reduced to only the portion of collateral that exceeds full collateral valuation of the debt assets that are borrowed from the blockchain. It is arguable who “owns” BTS that are locked up as collateral in BitAsset smart contracts. The authors of this BSIP hold the view that debt holders have received BitAssets in return for their collateral (which they may have sold on the market for more BTS), and thus have given up rights to the collateral itself, at least to the portion of collateral that represents the value of the BitAssets they have received. It is only logical that by exchanging the collateral for BitAssets they have given up the rights associated with the collateral, including its voting power.
```

Comment:
I still think the changes about voting should not be in this BSIP. Better discuss it in the scope of [BSIP-0024 - Locking Bitshares away as 'Bitshares Influence' for voting privileges on the BTS DEX](https://github.com/bitshares/bsips/blob/master/bsip-0024.md) and/or https://github.com/bitshares/bsips/issues/83.

Reply:
I don't think it falls under 24/83 since they propose a bitshares 'stake power' like in Steem instead of eliminating the vote weight of the <=1x CR. I do agree that a separate BSIP for this idea would be appropriate, I like it.

Reply:
The debt holders still should have the full voting power even if  full or not full collateral.

The reason is very simple, when another DEX have collateral mechanism of bts, and pledge a lot of bts, when price drop down, all the debt didn't get margin called in the market asap, so you tell me this DEX didn't have the voting power?

Reply:
> so you tell me this DEX or CEX didn't have the full voting power?

Perhaps we could encourage users to return BTS from CEX to the DEX? CEX voting with user funds (even proxying their vote weight) is not appropriate.

Reply:
Removed this point.